docs/attributes-registry/gen-ai.md

```diff
@@ -12,26 +12,27 @@ This document defines the attributes used to describe telemetry in the context o
 
```

Comment:
Should the sentence prior be updated to read, "This document defines the attributes used to describe telemetry in the context of Generative Artificial Intelligence (GenAI) requests and responses."?